packages/blockchain-wallet-v4-frontend/src/modals/index.tsx

```diff
@@ -123,6 +123,9 @@ const SimpleBuy = React.lazy(() => import('./SimpleBuy'))
 const Swap = React.lazy(() => import('./Swap'))
 const Withdraw = React.lazy(() => import('./Withdraw'))
 
+// BROKERAGE
+const Banks = React.lazy(() => import('./Brokerage/Banks'))
```

Comment:
No changed needed here, but we should probably wrap this suspense with an error boundry since there are so many components inside it.

Reply:
nah we separate logic here, so `Banks` are isolated now, they belong yo Brokerage but it's stand alone flyout, so we do not need a central index file for Brokerage and then from there do all nasty things and conditions.

Reply:
the whole app is wrapped in an Error boundary, this should be fine